mdgen/src/main/java/uk/gov/ida/mdgen/MetadataGenerator.java

```diff
@@ -94,6 +94,10 @@
     @CommandLine.Option(names = "--hsm-metadata-signing-label", description = "HSM Metadata key label")
     private String hsmMetadataKeyLabel = "private_key";
 
+    @CommandLine.Option(names = "--validity", description = "How many days the metadata is valid for (default 30 days)")
+    private Integer validity = 30;
```

Comment:
```suggestion
    private Integer validityDays = 30;
```
Call the param `validityDays` to avoid potential confusion?